### PR TITLE
Fel 822

### PR DESCRIPTION
**Pull request by nicolas-racchi**

### CODE REVIEW

File: frontend/src/components/projects/active-campaign/active-campaign-view.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ const ActiveCampaignsView = ({
         <>
           {pathname === "/active-campaigns" ? (
             <>
-              {campaigns.length > 0 ? (
-                <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
```
Comment (Knos01) [inline frontend/src/components/projects/active-campaign/active-campaign-view.tsx:55]: Why have you removed my changes here? That was for displaying only one column on mobile, does it work anyway?

File: frontend/src/pages/projects/active-campaigns.tsx [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+import { fakeCampaigns } from "../../../mockData";
 
 const ActiveCampaigns: NextPage = () => {
-  return <ActiveCampaignView firstTab="active" secondTab="participated" />;
+  const campaigns = useMemo(() => fakeCampaigns, []);
+
+  return (
+    <div className="grid max-w-4-xl mx-auto mt-24">
+      {campaigns?.length ? (
+        <div className="grid grid-cols-3 gap-4">
```
Comment (Knos01) [inline frontend/src/pages/projects/active-campaigns.tsx:13]: Does it display one column on mobile?

File: frontend/src/pages/projects/my-campaigns.tsx [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
+import { fakeMyCampaigns } from "../../../mockData";
 
 const MyCampaigns: NextPage = () => {
-  return <MyCampaignsView />;
+  const campaigns = useMemo(() => fakeMyCampaigns, []);
+
+  return (
+    <div className="grid max-w-4-xl mx-auto mt-24">
+      {campaigns?.length ? (
+        <div className="grid grid-cols-3 gap-4">
```
Comment (Knos01) [inline frontend/src/pages/projects/my-campaigns.tsx:13]: Same one column question here

File: frontend/src/components/projects/active-campaign/active-campaign-view.tsx (hunk 1 above)
Comment (nicolas-racchi) [reply frontend/src/components/projects/active-campaign/active-campaign-view.tsx:55]: good catch, fixed, thanks

**Review by Knos01 — APPROVED** [review]

**Pull request merged**